record/record_test.go

```diff
@@ -399,13 +399,16 @@ func TestBasicRecover(t *testing.T) {
 }
 
 func TestComplexRecover(t *testing.T) {
-	// The first record will be blockSize * 3 bytes long. Since each block has
-	// a 6 byte header, the first record will roll over into 4 blocks.
+	// The first record will be blockSize * 3 bytes long. Since each block has a
```

Comment:
I know that gofmt accepts either, but please leave the line breaking for multi-line comments alone, if possible. The lines are broken automatically by whatever my editor (vim) does when I hit "gq" (with tab-stop = 4).